src/nut.c

```diff
@@ -54,8 +54,11 @@ static nut_ups_t *upslist_head = NULL;
 static pthread_mutex_t read_lock = PTHREAD_MUTEX_INITIALIZER;
 static int read_busy = 0;
 
-static const char *config_keys[] = {"UPS"};
+static const char *config_keys[] = {"UPS","FORCESSL","VERIFYPEER","CAPATH"};
 static int config_keys_num = STATIC_ARRAY_SIZE(config_keys);
+static int force_ssl = 0; // Initialized to default of 0 (false)
```

Comment:
This isn't about initializing to 0 to ensure the value is zero from a programmatic standpoint. The default behavior if the config does not specify a value should be false (0). Setting it to 0 makes it clear that if it doesn't get changed by the config file, it will be 0 and thus equivalent to "false".